### PR TITLE
Dev: correct file location used for setting message rates

### PR DESCRIPTION
**Pull request by rmackay9**
This resolves some user confusion raised in this issue https://github.com/ArduPilot/ardupilot/issues/22361

### CODE REVIEW

**Review by peterbarker — APPROVED** [review]

File: dev/source/docs/mavlink-requesting-data.rst [hunk 1]
```diff
@@ -159,7 +159,7 @@ Specifying Message Rates in a File
 
 At boot ArduPilot will populate the initial message intervals from files found in either ROMFS or in the filesystem.
 
-On ChibiOS-based boards the SD card will be searched for specially-named files in the APM subdirectory.
+On ChibiOS-based boards (with more than 1MB of flash) the SD card will be searched for specially-named files in the top directory.
```
Comment (Hwurzburg) [inline dev/source/docs/mavlink-requesting-data.rst:162]: sorry I have never heard the root or root directory of the SD card called "top" directory
```suggestion
On ChibiOS-based boards (with more than 1MB of flash) the SD card will be searched for specially-named files in the  root directory, sometimes called top or base directory.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: dev/source/docs/mavlink-requesting-data.rst (hunk 1 above)
Comment (rmackay9) [reply dev/source/docs/mavlink-requesting-data.rst]: Yes, "root" is better than "top" so I've changed it.  txs!

**Comment by rmackay9** [comment]
@Hwurzburg,

Obviously no rush but maybe we can merge this now?

**Comment by Hwurzburg** [comment]
thanks

**Pull request merged**